### PR TITLE
regarding issue #11, added $REPOSITORY and $PROJECT 

### PR DESCRIPTION
**Pull request by uweschaefer**
also extended four Tests (mocking) and parameterized-build-hook.soy accordingly

### CODE REVIEW

File: src/main/java/com/kylenicholls/stash/parameterizedbuilds/item/GetQueryStringParameters.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package com.kylenicholls.stash.parameterizedbuilds.item;
+
+import com.google.common.base.Preconditions;
+
+public class GetQueryStringParameters {
```
Comment (KyleLNicholls) [inline src/main/java/com/kylenicholls/stash/parameterizedbuilds/item/GetQueryStringParameters.java]: Thanks for this! +10

**Comment by KyleLNicholls** [comment]
Eventually I want to clean up the tests a bit and add some more coverage around the areas commonly touched

**Comment by KyleLNicholls** [comment]
@uweschaefer I added you as a contributor. Feel free to push to the repo but any changes to master should go through pull requests. That's the only rule I have.
Thanks for bringing this plugin back to life.

**Comment by uweschaefer** [comment]
thanks.

**Pull request merged**